src/app/search/search-filters/simple.tsx

```diff
@@ -1,11 +1,19 @@
-import { tl } from 'app/i18next-t';
+import { t, tl } from 'app/i18next-t';
 import { DimItem } from 'app/inventory/item-types';
 import { isSunset } from 'app/utils/item-utils';
 import { BucketHashes } from 'data/d2/generated-enums';
 import { FilterDefinition } from '../filter-types';
 
 // simple checks against check an attribute found on DimItem
 const simpleFilters: FilterDefinition[] = [
+  {
+    keywords: 'adept',
+    description: tl('Filter.AdeptDescription'),
+    filter: () => (item: DimItem) =>
+      new RegExp(t('Filter.Adept')).test(item.name) ||
+      new RegExp(t('Filter.Harrowed')).test(item.name) ||
+      new RegExp(t('Filter.Timelost')).test(item.name),
+  },
```

Comment:
A good optimization would be to move the regex construction outside of the predicate -- right now this constructs three RegEx objects for every item in the inventory.

Reply:
Ah, that'd be pretty good, lol. I'll work on this tonight.

Thanks for bringing it to my attention~

Reply:
I was thinking something like
```
filter: () => {
      const regexes = [t('Filter.Adept'), t('Filter.Harrowed'), t('Filter.Timelost')].map(
        (pattern) => new RegExp(pattern)
      );
      return (item: DimItem) => regexes.some((re) => re.test(item.name));
    },
```
The `inWeapons` check in addition seems reasonable.

Reply:
Ah, that makes much more sense than what I had tried, haha.
I've gone ahead and updated the filter logic to what you were thinking as well as an `inWeapons` check.

It works as expected.